middleware/dnssec/README.md

```diff
@@ -26,11 +26,17 @@ TODO(miek): think about key rollovers, and how to do them automatically.
 ~~~
 dnssec [ZONES... ] {
     key file KEY...
+    cache_capacity CAPACITY
```

Comment:
I don't have a good sense to terms. Maybe `capacity` or `lru` to make it simple?